content/en/docs/languages/rust/_index.md

```diff
@@ -30,4 +30,4 @@ OpenTelemetry for Rust publishes the following crates:
 
 ## Further Reading
 
-- [Ecosystem](https://github.com/open-telemetry/opentelemetry-rust#ecosystem)
+- [Overview of crates](https://github.com/open-telemetry/opentelemetry-rust#overview-of-crates)
```

Comment:
This change was made via:

- https://github.com/open-telemetry/opentelemetry-rust/pull/1581